runbotics/runbotics-orchestrator-ui/src/src-app/views/users/DeleteUserDialog.tsx

```diff
@@ -37,20 +38,23 @@ const StyledButton = styled(Button)`
 interface DeleteUserDialogProps {
     open?: boolean;
     onClose: () => void;
+    closeEditDialog: () => void;
```

Comment:
you should not include this prop `closeEditDialog`
you already have `onClose` so just close your modal in this callback